in-depth/submission_guidelines.md

```diff
@@ -86,6 +86,10 @@ Heading
 
 : The name of your theme or plugin.
 
+View
+
+: Choose between the image oriented `jumbovision` layout, or the text-oriented `card` layout.  The default for Themes is `jumbovision`, while plugins use `card`.  This option only needs to be specified if you'd like to use the opposite layout.
```

Comment:
Maybe. The Contribute tile says "Learn More", so we could also go with that.

Between the two, which do you like better? "More Info" or "Learn More"?

We could also go with something like "Visit Page". Hmm. :thinking:

And I also like "View on GitHub", because these days I don't take a project seriously unless they're on GitHub. :laughing:

"Visit us on SourceForge"? No thanks. :-1: